src/org/omegat/core/team2/impl/GITRemoteRepository2.java

```diff
@@ -126,7 +126,10 @@ public void init(RepositoryDefinition repo, File dir, ProjectTeamSettings teamSe
             CloneCommand c = Git.cloneRepository();
             c.setURI(repositoryURL);
             c.setDirectory(localDirectory);
-            c.setTimeout(TIMEOUT);
+            // BUGS#1075 - When set timeout, JSch expect to check result of setEnv('GIT_PROTOCOL=version=2')
+            // and throw exception when failed, but it should ignored.
+            // It is JSch bug but we avoid it by not setting timeout here.
+            // c.setTimeout(TIMEOUT);
```

Comment:
A fix is requested on mwiede/jsch project.
https://github.com/mwiede/jsch/pull/92

Reply:
Thanks for taking this upstream. I see they've merged the fix.

Reply:
The fix is merged.
https://github.com/mwiede/jsch/pull/90

Reply:
The fix is released and worked.